recipes/mdsplus/mdsdcl_Makefile.in.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/mdsdcl/Makefile.in b/mdsdcl/Makefile.in
+index 9772b5c7c..3cc1d2e33 100644
+--- a/mdsdcl/Makefile.in
++++ b/mdsdcl/Makefile.in
+@@ -41,13 +41,13 @@ install: $(libdir) $(bindir)
+ 		@RANLIB@ $@
+ 
+ mdsdclThreadSafe.o: mdsdclThreadSafe.c
+-	$(COMPILE.c) $(CFLAGS) -c -I/usr/include/libxml2 $^
++	$(COMPILE.c) $(CFLAGS) -c -I$(PREFIX)/include/libxml2 $^
```

Comment:
Instead of patching, do `export CFLAGS="$CFLAGS -I${PREFIX}/include/libxml2` in  `build.sh`